src/main/java/org/broadinstitute/consent/http/db/ElectionDAO.java

```diff
@@ -87,10 +87,19 @@ void bulkCancelOpenElectionByReferenceIdAndType(@Bind("electionType") String ele
             + " where e.referenceId = :referenceId and lower(e.electionType) = lower(:type) and e.version = (select MAX(version) from election where referenceId = :referenceId)")
     Election getElectionWithFinalVoteByReferenceIdAndType(@Bind("referenceId") String referenceId, @Bind("type") String type);
 
-    @SqlQuery("select e.electionId,  e.datasetId, v.vote finalVote, e.status, e.createDate, e.referenceId, v.rationale finalRationale, v.createDate finalVoteDate, "
-            + " e.lastUpdate, e.finalAccessVote, e.electionType,  e.dataUseLetter, e.dulName, e.archived, e.version  from election e"
-            + " left join vote v on v.electionId = e.electionId and lower(v.type) = 'chairperson' "
-            + " where  e.electionId = :electionId")
+    @SqlQuery("SELECT distinct "
+          + "    e.electionid, e.datasetid, v.vote finalvote, e.status, e.createdate, "
+          + "    e.referenceid, v.rationale finalrationale, v.createdate finalvotedate, "
+          + "    e.lastupdate, e.finalaccessvote, e.electiontype,  e.datauseletter, e.dulname, "
+          + "    e.archived, e.version "
+          + "FROM election e "
+          + "LEFT JOIN vote v ON v.electionid = e.electionid AND "
+          + "    CASE "
+          + "        WHEN LOWER(e.electiontype) = 'dataaccess' THEN 'final' "
+          + "        WHEN LOWER(e.electiontype) = 'dataset' THEN 'data_owner' "
```

Comment:
Nothing wrong with the code, I'm just curious about the context of the data_owner vote. When I think of the voting process, the things that come to mind are the chair vote, the member votes, and the final vote, but not the data_owner vote (but it's being used here as a backup check on final).

Reply:
The data owner vote is probably something none of us has actually seen. It's a separate console that shows up just for data owners when there is a DO election that involves one of their datasets. In that case, the final vote belongs to them.